### PR TITLE
Add code block style

### PR DESCRIPTION
**Pull request by csellis**
Add styling to set off <code> in Show Notes

Screenshot from 2019-10-03 21-23-34

I couldn't find the Syntax.fm style guide. I thought it looked decent though. #407

### CODE REVIEW

File: styles/_show.styl [hunk 1]
```diff
@@ -102,4 +102,19 @@
     color lighten(black, 20%)
     border-bottom 1px solid yellow
     text-decoration none
+  code
+    background: #f4f4f4;
+    border: 1px solid #ddd;
+    border-left: 3px solid #f36d33;
```
Comment (416serg) [inline styles/_show.styl]: probably a good idea to use the `yellow` from https://github.com/wesbos/Syntax/blob/master/styles/_variables.styl

Comment (csellis) [reply styles/_show.styl]: That sounds great. @Wes is particular about his yellows.

Comment (416serg) [reply styles/_show.styl]: yeah he is 😅

Comment (csellis) [reply styles/_show.styl]: @416serg I got around to adding the yellow

**Comment by wesbos** [comment]
this is awesome thanks!

**Pull request merged**